README.md

```diff
@@ -94,3 +81,25 @@ docker run -it \
       "id":1
     }' -s | jq
   ```
+
+## Usage remarks
+
+Currently Potterhat behaves as a dumb relay to Ethereum nodes, meaning that discrepancies
+between implementations like Geth v.s. Parity are not handled. It is up to the user to make sure
```

Comment:
if that's the case currently... is there a way to pass down the information what type of client made the response?

Reply:
By making an `web3_clientVersion` call? 😂 

In all seriousness, we could utilize these two http headers?

```
Server: Geth/v1.8.27-stable-4bcc0a37/linux-amd64/go1.11.9`
Via: Potterhat/v0.1.0
```

Another way is by passing down in all response body but I don't think it will be a good idea to mix it with the actual node response.

WDYT @InoMurko @T-Dnzt @sirn?

Reply:
Issue created: https://github.com/omisego/potterhat/issues/31